### PR TITLE
Misc survivor gear

### PR DESCRIPTION
**Pull request by ArcanErasmus**
#### Summary
```SUMMARY: Content "Adds new survivor gear: atomic headlamp, heelys, fingerless gloves"```

#### Purpose of change
When playing, I ran into a couple of cases where I really wanted to make an item, but it didn't exist. Looking at existing items, and existing recipes, I tried to come up with both realistic and balanced new items to fit some fun niches.

#### Describe the solution
Added 3 new items:

1. Fingerless Survivor Gloves. These are just Light Survivor Gloves with the fingers cut off. The main use is to allow access to claw-based attacks, whether mutation or bionic. Currently, there are no nicely armored fingerless gloves that are well-made and require solid skills.
2. Survivor Heelys. Similar to the previous item, these are based off Light Survivor Boots, except crafted with heelys. This came up in a discord discussion a while back, and was then forgotten, but the idea tickled me enough to play with. They are fun and armored! To compensate for basing them off heelys, instead of boots, the protection values, thickness value, and warmth are lower.
3. Atomic Headlamp. Seeing as we can craft headlamps, and we can craft atomic lamps, why not have an infinite headlamp? The recipe and stats are based off the Survivor Headlamp. Even incorporating a lens to focus it, the light output is still significantly lower than that of even a regular headlamp. While it makes for a fuss-free source of more than enough light to traverse the night, the downside is that it is worn over your headgear, and thus is quite likely to be hit if you take a shot to the head. And if it gets damaged, you might get to bathe in more of its glow than you wanted...

#### Describe alternatives you've considered
- Lowering the protective values on fingerless survivor gloves
- Higher or lower light level on the atomic headlamp
- Removing the `DURABLE` flag from the atomic headlamp
- Increasing the protective values of the survivor heelys, but adding encumbrance
- Lowering the activation time on the survivor heelys to the same as regular heelys

#### Testing
All recipes craft and uncraft successfully.

Light output of atomic headlamp is lower than a regular headlamp, but still quite usable.

Gloves allow natural attacks.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
Make sure to blacklist these in the no survivor gear mod.

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -590,6 +590,48 @@
...
+    "category": "clothing",
+    "symbol": "[",
+    "color": "blue",
+    "looks_like": "survivor_light_on",
+    "name": "atomic headlamp",
+    "description": "This is a custom made reinforced headlamp powered by the magic of nuclear decay, focused for more usable brightness.  The adjustable strap allows it to be comfortably worn on your head or attached to your helmet.  Use it to close the cover and hide the light.",
+    "price": 4000,
+    "material": [ "steel", "plastic" ],
+    "flags": [ "LIGHT_200", "LEAK_DAM", "RADIOACTIVE", "DURABLE_MELEE", "OVERSIZE", "BELTED", "ALLOWS_NATURAL_ATTACKS" ],
+    "weight": "900 g",
```
Comment (Rivet-the-Zombie) [inline data/json/items/tool_armor.json]: Does this thing's weight add up properly? It seems awfully lightweight.

Comment (ArcanErasmus) [reply data/json/items/tool_armor.json]: Actually, almost! I forgot to replace my eyeballed number with actual math. The total weight is 1kg, but the volume should be much larger, like 1.5L.

**Comment by kevingranade** [comment]
The rationale for the survivor heeleys evades me, how is this different from adding kevlar patches to regular heeleys?
That aside adding kevlar patches to a sneaker also doesn't make much sense, it doesn't have the requisite thickness and backing to meaningfully spread out the force from attacks.

**Comment by ArcanErasmus** [comment]
my idea is that you would borrow the sole and maybe heel backing from the heelys, and the rest is basically handcrafted. I could change it to start with survivor boots as well, where you would just be implanting the wheel assembly into the boots.

The differences from just reinforcing heelys are adding water protection, durability, and ability for further reinforcement.

**Comment by Beneficial-Insurance** [comment]
Survivor heeleys sound like complete and total BS to me

**Comment by kevingranade** [comment]
You need to fix the heely boot description then, it claims it's an up-armored sneaker instead of a boot with a heely sole glued to it. Also bump up the weight a bit.
Also in that case, why aren't you using copy-from in the definition?

**Comment by ArcanErasmus** [comment]
Considering feedback, I've gone ahead and removed the survivor heelys.

**Pull request merged**